### PR TITLE
HLM-299 | Revoke VC - API for revocation

### PR DESCRIPTION
**Pull request by vishwajeet-eGov**

### CODE REVIEW

**Comment by egov-joy** [comment]
@vishwajeet-eGov the story mentions maintaining a revocation list. That has not been done.
cc @prasanna-egov

File: backend/vc-certification-service/src/routes/certificate.route.js [hunk 1]
```diff
@@ -7,5 +7,6 @@ const router = express.Router();
 router.post(`/certify/:entityType`, authMiddleware, certificateController.createCertificate)
 router.get(`/certificate/:entityName/:certificateId`, authMiddleware, certificateController.getCertificate)
 router.put(`/certify/:entityName/:certificateId`, authMiddleware, certificateController.updateCertificate);
+router.delete(`/certificate/:entityName/:certificateId`, authMiddleware, certificateController.deleteCertificate);
```
Comment (prasanna-egov) [inline backend/vc-certification-service/src/routes/certificate.route.js:10]: Can you also update swagger.json with the details for the API.

**Review by prasanna-egov — APPROVED** [review]

File: backend/vc-certification-service/certification-service-swagger.yml [hunk 2]
```diff
@@ -125,6 +125,38 @@ paths:
           description: Internal server error
           schema:
             $ref: '#/definitions/Error'
+  /v1/certificate/{entityName}/{certificateId}:
```
Comment (egov-joy) [inline backend/vc-certification-service/certification-service-swagger.yml:128]: We need to standardise the naming of the param in the APIs. Some places have it as entityId and some have it as certificateId

Comment (vishwajeet-eGov) [reply backend/vc-certification-service/certification-service-swagger.yml:128]: Updated the Swagger.

Comment (egov-joy) [reply backend/vc-certification-service/certification-service-swagger.yml:128]: @prasanna-egov the PUT operation uses `entityId` in the swagger definition whereas the the other operations use certificateId. Can we standardize?

Comment (prasanna-egov) [reply backend/vc-certification-service/certification-service-swagger.yml:128]: True we should call it certificateId

Comment (prasanna-egov) [reply backend/vc-certification-service/certification-service-swagger.yml:128]: @vishwajeet-eGov could you update route as well as swagger.

Comment (vishwajeet-eGov) [reply backend/vc-certification-service/certification-service-swagger.yml:128]: Done

**Review by egov-joy — APPROVED** [review]

**Review by prasanna-egov — APPROVED** [review]

**Pull request merged**